### PR TITLE
Plains biome generation fix

### PR DESCRIPTION
**Pull request by fluffyloafie**
- Add the flowers
- Add the grass
- Have less trees
- Is flatter

### CODE REVIEW

**Review by STMiki — APPROVED** [review]

File: cubic-server/generation/generator.cpp [hunk 1]
```diff
@@ -33,9 +33,11 @@ generation::Generator::GenerationNoise generation::Generator::getNoise(positionT
     noise.noise2D.continentalness = _noiseMaker.octave2D_11(_x, _z, octaves);
     // noise.noise2D.erosion = _noiseMaker.octave2D_11(_x, _z, octaves);
     // noise.noise2D.peaksAndValley = _noiseMaker.octave2D_11(_x, _z, octaves);
-    noise.noise2D.weirdness = _noiseMaker.octave2D_11(_x, _z, octaves);
-    noise.noise2D.trees = _noiseMaker.octave2D_11(x * 0.5, z * 0.5, 1);
+    noise.noise2D.weirdness = _noiseMaker.octave2D_11(_x * 0.5, _z * 0.8, 6);
```
Comment (Trompettesib) [inline cubic-server/generation/generator.cpp:36]: please add a comment to explain the magic numbers

File: cubic-server/generation/overworld.cpp [hunk 2]
```diff
@@ -52,8 +52,8 @@ BlockId generation::Overworld::getBlock(positionType x, positionType y, position
 
     //! NEW NEW
     auto noise = getNoise(x, y, z);
-    int heightOffset = 100;
-    int surfaceLevel = heightOffset + noise.noise2D.continentalness * 20;
+    int heightOffset = 70;
+    int surfaceLevel = heightOffset + noise.noise2D.continentalness * 5;
```
Comment (Trompettesib) [inline cubic-server/generation/overworld.cpp:56]: please add a comment to explain the magic value

File: cubic-server/registry/Biome.hpp [hunk 3]
```diff
@@ -110,13 +110,13 @@ inline void registry::setupDefaultsBiome(registry::Biome &registry)
         .name("cubsrv:my_super_cool_biome_lol_haha")
         .id(1)
         .precipitation("none")
-        .temperature(0.8)
-        .downfall(0.4)
+        .temperature(0)
```
Comment (Trompettesib) [inline cubic-server/registry/Biome.hpp:113]: Why did you change this biome knowing that it does not appear on generation ?

**Review by Trompettesib — COMMENTED** [review]
Good, but lack some comments
Da chikin still spawn in the sky
Why we only have one biome now ?

**Review by emneo-dev — APPROVED** [review]

File: cubic-server/registry/Biome.hpp (hunk 3 above)
Comment (NekoSelena) [reply cubic-server/registry/Biome.hpp:113]: So I can see that it, in fact, does not appear
